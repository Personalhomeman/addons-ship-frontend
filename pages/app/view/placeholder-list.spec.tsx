import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import PlaceholderList from './placeholder-list';

describe('PlaceholderList', () => {
  it('renders correctly', () => {
    const tree = shallowToJson(shallow(<PlaceholderList />));
    expect(tree).toMatchSnapshot();
  });
});
