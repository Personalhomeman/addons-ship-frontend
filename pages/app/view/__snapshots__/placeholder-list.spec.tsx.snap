// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlaceholderList renders correctly 1`] = `
<Flex
  container={true}
  direction="vertical"
  margin="x16"
>
  <Skeleton
    active={true}
    color="white"
    maxWidth="60rem"
  >
    <Card
      direction="vertical"
      gap="x10"
    >
      <CardContent
        _padding="x4"
      >
        <SkeletonBox
          backgroundColor="gray-2"
          height="5rem"
          margin="x1"
        />
        <SkeletonBox
          absolute={true}
          backgroundColor="gray-2"
          borderColor="gray-3"
          borderRadius="x2"
          height="6rem"
          style={
            Object {
              "left": "2rem",
              "top": "2rem",
            }
          }
          width="6rem"
        />
      </CardContent>
      <CardContent
        padding="x4"
      >
        <SkeletonBox
          backgroundColor="gray-2"
          height="2rem"
          margin="x3"
          width="18rem"
        />
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </CardContent>
    </Card>
    <Base
      margin="x16"
    >
      <SkeletonBox
        backgroundColor="gray-2"
        height="2.5rem"
        margin="x2"
        width="14rem"
      />
      <Flex
        direction="vertical"
        gap="x1"
        margin="x6"
      >
        <Flex
          alignChildrenHorizontal="between"
          direction="horizontal"
        >
          <SkeletonBox
            backgroundColor="gray-2"
            height="2rem"
            margin="x1"
            width="16rem"
          />
          <SkeletonBox
            backgroundColor="gray-2"
            height="1.5rem"
            margin="x1"
            width="10rem"
          />
        </Flex>
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </Flex>
      <Divider
        color="gray-2"
        direction="horizontal"
        width="0.125rem"
      />
      <Flex
        direction="vertical"
        gap="x1"
        margin="x6"
      >
        <Flex
          alignChildrenHorizontal="between"
          direction="horizontal"
        >
          <SkeletonBox
            backgroundColor="gray-2"
            height="2rem"
            margin="x1"
            width="16rem"
          />
          <SkeletonBox
            backgroundColor="gray-2"
            height="1.5rem"
            margin="x1"
            width="10rem"
          />
        </Flex>
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </Flex>
      <Divider
        color="gray-2"
        direction="horizontal"
        width="0.125rem"
      />
      <Flex
        direction="vertical"
        gap="x1"
        margin="x6"
      >
        <Flex
          alignChildrenHorizontal="between"
          direction="horizontal"
        >
          <SkeletonBox
            backgroundColor="gray-2"
            height="2rem"
            margin="x1"
            width="16rem"
          />
          <SkeletonBox
            backgroundColor="gray-2"
            height="1.5rem"
            margin="x1"
            width="10rem"
          />
        </Flex>
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </Flex>
      <Divider
        color="gray-2"
        direction="horizontal"
        width="0.125rem"
      />
      <Flex
        direction="vertical"
        gap="x1"
        margin="x6"
      >
        <Flex
          alignChildrenHorizontal="between"
          direction="horizontal"
        >
          <SkeletonBox
            backgroundColor="gray-2"
            height="2rem"
            margin="x1"
            width="16rem"
          />
          <SkeletonBox
            backgroundColor="gray-2"
            height="1.5rem"
            margin="x1"
            width="10rem"
          />
        </Flex>
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </Flex>
      <Divider
        color="gray-2"
        direction="horizontal"
        width="0.125rem"
      />
      <Flex
        direction="vertical"
        gap="x1"
        margin="x6"
      >
        <Flex
          alignChildrenHorizontal="between"
          direction="horizontal"
        >
          <SkeletonBox
            backgroundColor="gray-2"
            height="2rem"
            margin="x1"
            width="16rem"
          />
          <SkeletonBox
            backgroundColor="gray-2"
            height="1.5rem"
            margin="x1"
            width="10rem"
          />
        </Flex>
        <SkeletonBox
          backgroundColor="gray-2"
          height="1rem"
          margin="x1"
        />
      </Flex>
      <Divider
        color="gray-2"
        direction="horizontal"
        width="0.125rem"
      />
    </Base>
  </Skeleton>
</Flex>
`;
